packages/components/src/date-time/date/index.tsx

```diff
@@ -303,7 +303,8 @@ function Day( {
 		// isFocusAllowed is not a dep as there is no point calling focus() on
 		// an already focused element.
 	}, [ isFocusable ] );
-
+	const shiftedDay = new Date( day );
+	shiftedDay.setHours( 15 );
```

Comment:
Can you explain a bit more about this technique? Dates have many nuances and would like to understand more about it.

Would similar changes be needed elsewhere or this could resolve this issue for the whole component?